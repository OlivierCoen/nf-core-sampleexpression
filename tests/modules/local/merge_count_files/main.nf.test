nextflow_process {

    name "Test Process MERGE_COUNT_FILES"
    script "modules/local/merge_count_files/main.nf"
    process "MERGE_COUNT_FILES"
    tag "merge_count_files"

    test("Should run without failures") {

        when {

            process {
                """
                ch_csv_files = Channel.fromPath( '$baseDir/tests/input/merge_count_files/*_renamed.csv', checkIfExists: true).toList()
                input[0] = ch_csv_files
                """
            }
        }

        then {
            assert process.success
        }

    }

}
