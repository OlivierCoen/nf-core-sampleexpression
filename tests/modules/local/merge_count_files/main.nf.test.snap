{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "merged.csv:md5,6e3bc6b34e05d88f83a4c0c209acb2f5"
                ],
                "csv": [
                    "merged.csv:md5,6e3bc6b34e05d88f83a4c0c209acb2f5"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-10-16T00:58:16.49303062"
    }
}